src/main/java/ca/wescook/nutrition/modules/witchery/events/WitcheryEventHandler.java

```diff
@@ -0,0 +1,35 @@
+package ca.wescook.nutrition.modules.witchery.events;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.PlayerCapabilities;
+import net.minecraftforge.event.entity.living.LivingEvent.LivingUpdateEvent;
+
+import com.emoniph.witchery.common.ExtendedPlayer;
+
+import ca.wescook.nutrition.data.NutrientManager;
+import ca.wescook.nutrition.data.PlayerDataHandler;
+import ca.wescook.nutrition.modules.witchery.WitcheryHelper;
+import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+
+public class WitcheryEventHandler {
+
+    public static final PlayerCapabilities genericPlayerCapabilities = new PlayerCapabilities();
+
+    @SubscribeEvent
+    public void livingTick(LivingUpdateEvent event) {
+        if (!WitcheryHelper.isActive()) {
+            return;
+        }
+
+        if ((event.entity instanceof EntityPlayer player)) {
+            NutrientManager nutritionOld = PlayerDataHandler.getForPlayer(player); // Get old nutrition
+            NutrientManager nutritionNew = new NutrientManager(nutritionOld.get()); // Create new nutrition
```

Comment:
Since this is a ticking event, maybe better to move these inside the inner conditional so the allocations only happen when the player is indeed a vampire.